README.md

```diff
@@ -1,10 +1,8 @@
 # Introduction
-GLUECOSE provides an interoperability framework for various COSE libraries.
-At the same time, it is a single place where one can find with ease, which COSE library
-is best suited for ones project specific needs.
+GLUECOSE provides an interoperability framework for various [CBOR Object Signing and Encryption](https://datatracker.ietf.org/doc/html/rfc8152) (COSE) libraries. Additionally, it provides a single place where developers can view the features of various COSE libraries, and thus select a library that is best suited for a project's specific needs.
 
 # Motivation
-COSE has a very wide feature spectrum and is quite complex. This makes it difficult for library implementors to test and verify their libraries. Also, many implementors choose to implement a subset of the full COSE specification. This makes it difficult for developers to know which COSE library to select given their project's needs.
+The COSE signing envelope format, which is broadly applicable from small devices to large-scale server environments, attempts to keep the flavor of [Javascript Object Signing and Encryption](https://datatracker.ietf.org/wg/jose/documents/) (JOSE) specifications ([JWS](https://datatracker.ietf.org/doc/rfc7515/), [JWE](https://datatracker.ietf.org/doc/rfc7516/), [JWK](https://datatracker.ietf.org/doc/rfc7517/), [JWA](https://datatracker.ietf.org/doc/rfc7518/)) while providing additional features and flexibility. To meet requirements for device reach and flexibility, COSE has a very wide feature set and is quite complex. This makes it difficult for library implementors to test and verify their libraries. Also, many implementors choose to implement a subset of the full COSE specification. This makes it difficult for developers to know which COSE library to select given their project's needs.
```

Comment:
To meet requirements for device reach and flexibility,

This line can be slightly better re-worded as:
To meet the requirements for a broad range of use cases (Devices, Edge or Servers)  and flexibility, COSE offers a wide feature set and is quite complex.  This makes it difficult for library implementers to test and verify their libraries. Also, many implementers choose to implement a subset of the full COSE specification.
This makes it difficult for application developers to know which COSE library to select given their project needs.

Reply:
Incorporated. Thanks Yogesh.